backend/controllers/userDetails.js

```diff
@@ -17,7 +17,22 @@ const userDetailsFactory = (
 
     const searchResultsUrl = req.session.searchResultsUrl ? req.session.searchResultsUrl : `${searchUrl}/results`
 
-    const [user, roles, groups] = await getUserRolesAndGroupsApi(res.locals, username, hasMaintainDpsUsersAdmin)
+    const [user, roles, groups, assignableGroups] = await getUserRolesAndGroupsApi(
```

Comment:
not that happy about this, looking really messy with `groupsMaintainGroupManager` and `groupsNotGroupManager`.
Suggest approving as is and we can look at rewriting tomorrow